### DISCUSSION TITLE
How to call ui.table methods with return values in NiceGUI 1.0

### DISCUSSION THREAD

**Discussion by johancj** [zauberzeug/nicegui#229] [Q&A] [answered]
Hi, how can I use methods in ui.table with a return value in nicegui version 1.0?

I would really appreciate an example like in https://github.com/zauberzeug/nicegui/discussions/150, but with for example the function ```getSelectedRows```.  https://github.com/zauberzeug/nicegui/discussions/150 is very similar to my case, but not applicable for nicegui v1.0

**Comment by falkoschindler** [comment]
Currently this doesn't seem possible with NiceGUI 1.0. But I created issue https://github.com/zauberzeug/nicegui/issues/230 for implementing this feature.

**Comment by falkoschindler** [comment, marked as answer]
As discussed in #230 the next release will introduce a new possibility to get return values from Ag Grid API calls:

```python
result = await ui.run_javascript(f'getElement({table.id}).gridOptions.api.getSelectedRows()')
```
